components/Numbers.tsx

```diff
@@ -56,7 +58,7 @@ export function Numbers({ color, info }: Props) {
               {info.speakers}
             </h1>
             <h2 className="pb-2 lg:pb-4 font-bold text-lg lg:text-3xl text-black">
-              {'Speakers' + (info.lines > 1 ? 's' : '')}
+              {"Speakers" + (info.lines > 1 ? "s" : "")}
```

Comment:
I made a typo here, it will evaluate to Speaker**ss**, we need to fix it, sorry for this @mehdi76sy :( cc @Tuhaj